src/main/java/games/strategy/engine/framework/startup/login/ClientLogin.java

```diff
@@ -17,6 +28,7 @@
   public static final String ENGINE_VERSION_PROPERTY = "Engine.Version";
   public static final String JDK_VERSION_PROPERTY = "JDK.Version";
   public static final String PASSWORD_PROPERTY = "Password";
+  public static final String ENCRYPTED_PASSWORD_PROPERTY = "RSA Encrypted Password";
```

Comment:
It's probably the opposite, I completely overlooked that we can use the password as symmetric key. If we use it as symmetric key a man-in-the-middle attack is impossible, so this is the better choice!